app/src/main/java/grinnell/appdev/edu/lyles/slidingtab/MenuTabColorizer.java

```diff
@@ -0,0 +1,24 @@
+package grinnell.appdev.edu.lyles.slidingtab;
+
+import android.content.Context;
+import android.support.v4.content.ContextCompat;
+
+import grinnell.appdev.edu.lyles.R;
+
+/**
+ * Created by Mattori on 9/29/16.
+ */
+
+public class MenuTabColorizer implements SlidingTabLayout.TabColorizer {
+
+    private Context mContext;
+
+    public MenuTabColorizer(Context ctxt) {
+        mContext = ctxt;
+    }
+
+    @Override
+    public int getIndicatorColor(int pos) {
+        return ContextCompat.getColor(mContext, R.color.tabsScrollColor);
```

Comment:
Use a static import for `ContextCompat` to keep the call short and clean. So `import android.support.v4.content.ContextCompat;` will now look like this: `import static android.support.v4.content.ContextCompat.getColor;`